tests/test_basic.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+
+from varianter_cit import Cit
+
+
+class Basic(unittest.TestCase):
+
+    def test_interface(self):
+        input_data = [2, 2, 2, 2, 6, 2, 2, 2, 2, 2, 2, 2, 2,
+                      2, 2, 2, 2, 3, 2, 2, 2, 2, 2, 2, 2, 2, 2]
+        t_value = 2
+        constraints = ['0 != 1 || 1 != 0',
+                       '4 != 4 || 5 != 1',
+                       '4 != 5 || 5 != 1',
+                       '0 != 1 || 1 != 0',
+                       '3 != 0 || 4 != 4',
+                       '3 != 1 || 4 != 5',
+                       '1 != 0 || 4 != 0',
+                       '0 != 1 || 3 != 0',
+                       '1 != 1 || 4 != 3',
+                       '1 != 1 || 2 != 0',
+                       '3 != 0 || 5 != 0',
+                       '3 != 1 || 5 != 0',
+                       '3 != 0 || 5 != 0',
+                       '0 != 0 || 4 != 2 || 5 != 1',
+                       '0 != 1 || 1 != 0 || 3 != 1']
```

Comment:
There will be a problem with [Parse pull request](https://github.com/avocado-framework/cit-ng/pull/10) because Solver works with different representation of constraints than array of strings. For now, Solver has functionality "create_constraint_array " which can parse those array of strings, but this functionality is move to "pars_file" in [Parse pull request](https://github.com/avocado-framework/cit-ng/pull/10). I don't know when we should solve this problem? Should we solve it now or when we will be merging [Parse pull request](https://github.com/avocado-framework/cit-ng/pull/10)?

Reply:
Oh, you're right.  I expect your PR to be merged "really soon" (as soon as you have an updated version), but this portraits an interesting and possibly recurring issue in distributed development: if one sends a proposal that matches the current status of a code base, in theory, it should not worry about other in-progress work.

Sometimes the maintainer will be left with two equally correct pull requests, and will have to handle the conflicts at merge time.

So, your point is valid, but as an exercise, I'll merge this (that maps the current status), and then you can solve the conflicts (drop the test code that will become outdated) as part of your v2 (because then it should pass the tests).